CHANGELOG.md

```diff
@@ -2,6 +2,109 @@
 
 ## Unreleased
 
+The 2.0 release contains significant breaking changes that will most likely affect all users,
+please be sure to read the migration notes below carefully.
+
+Note: 1.32.x will be security patched for at least 6 months in case some of the changes below are
+too disruptive to adopt right away.
+
+### Migration notes
```

Comment:
Should we perhaps name the header "Breaking changes"? With some extra ⚠️  emojis as well, so that it catches the eye